src/envo/envo-edit.owl

```diff
@@ -12800,16 +12771,16 @@ SubClassOf(<http://purl.obolibrary.org/obo/ENVO_00001999> ObjectSomeValuesFrom(<
 
 # Class: <http://purl.obolibrary.org/obo/ENVO_00002000> (slope)
 
-AnnotationAssertion(Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> "MA:ma"^^xsd:string) <http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/ENVO_00002000> "A solid astronomical body part which is part of the planetary surface between the peak of an elevation or the bottom of a depression and  relatively flat surrounding land."^^xsd:string)
+AnnotationAssertion(Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> "MA:ma"^^xsd:string) <http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/ENVO_00002000> "A solid astronomical body part which is part of the planetary surface between the peak of an elevation or the bottom of a depression and relatively flat surrounding land."^^xsd:string)
```

Comment:
Isn't the essential feature of a slope the fact that is has a surface elevation gradient greater than something like 5 degrees?  This definition would include flat terraces between a peak and a valley as slopes.